almalinux_build_node.py

```diff
@@ -93,12 +97,16 @@ def init_working_dir(config):
                 #   rm: cannot remove 'test/test.txt': Permission denied
                 rm_sudo(os.path.join(working_dir, name))
     else:
-        logging.debug('creating the {0} working directory'.
-                      format(config.working_dir))
+        logging.debug(
+            'creating the {0} working directory'.format(config.working_dir)
```

Comment:
Not critical, but I'd prefer to use either %-syntax or f-strings when logging stuff - same applies to others around

Reply:
yeah, i thought the same but it's everywhere in the old code. maybe i should create a separate task to get rid of this kind of formatting all over the albs?